pandas/core/arrays/timedeltas.py

```diff
@@ -355,6 +355,31 @@ def to_pytimedelta(self):
         """
         return tslibs.ints_to_pytimedelta(self.asi8)
 
+    def to_timedelta64(self):
+        """
+        Return numpy array with timedelta64[ns] dtype
+
+        Returns
+        -------
+        ndarray[timedelta64[ns]]
+
+        Notes
+        -----
+        This returns a view on self, not a copy.
+
+        See also
+        --------
+        Timedelta.to_timedelta64
+        """
+        return self.asi8.view('m8[ns]')
+
+    @property
+    def asm8(self):
```

Comment:
do we need both `to_timedelta64` and `asm8` ?

Reply:
If we were starting fresh I would say no, but as it is I think consistency with the scalar type is really worthwhile.  (and to your point aboe DatetimeIndex should have a `to_datetime64` method)